client/src/components/FlagshipMenu/Flagship.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import { Route, Link } from 'react-router-dom'
+import './Flagship.css'
+import Icons from '../Icons/FAIcon'
+
+const Menu = (props) => {
```

Comment:
Can you name the component the same as the file and directory? You want all of those to be consistent. Right now you have FlagShipMenu, Flagship, and Menu